lib/supa/command.rb

```diff
@@ -16,8 +16,12 @@ def represent
     private
 
     def value
-      return @representer.send(@options[:modifier], apply_render_flags(raw_value)) if @options[:modifier]
-      apply_render_flags(raw_value)
+      return @value if defined?(@value)
+      @value = modifier ? @representer.send(modifier, apply_render_flags(raw_value)) : apply_render_flags(raw_value)
+    end
```

Comment:
I noticed that we call `value` more than once in some commands like `object` or `collection` (now also `attribute`).